README.md

```diff
@@ -44,53 +44,51 @@ instructions using GitPod.
 
 ## Install
 
-### Using NPM
+The following instructions show how the latest community release may be
+installed. The instructions should be adapted if you want to use the latest
+development release or any other older release.
+
+### Install via NPM
 
 Using the [hoprd npm package][6]:
 
 ```sh
 mkdir MY_NEW_HOPR_TEST_FOLDER
 cd MY_NEW_HOPR_TEST_FOLDER
-npm install @hoprnet/hoprd@1.72
+yarn add install @hoprnet/hoprd@1.72
```

Comment:
lets use npm for Kiautchou and revisit this in Moscow

Reply:
done